config/destination.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/conduitio-labs/conduit-connector-firebolt/config/validator"
+)
+
+// Destination holds destination-related configurable values.
+type Destination struct {
+	Common
+}
+
+// ParseDestination attempts to parse plugins.Config into a Destination struct.
+func ParseDestination(cfg map[string]string) (Destination, error) {
+	common, err := ParseCommon(cfg)
+	if err != nil {
+		return Destination{}, fmt.Errorf("parse common config: %w", err)
+	}
+
+	destination := Destination{common}
+
+	if err := validator.Validate(destination); err != nil {
```

Comment:
Is it necessary to validate `destination` if it contains only a `common`? Or was this added in case the structure was expanded?

Reply:
Yeah, deleted the call